server/api/themoviedb/index.ts

```diff
@@ -368,17 +378,18 @@ class TheMovieDb extends ExternalAPI {
           page,
           include_adult: includeAdult,
           language,
-          with_release_type: '3|2',
```

Comment:
Why are we removing this?

Reply:
It was removing valid results (e.g., items which only have a digital release).  I'm unsure why this was added in the first place?

Reply:
It ignores the region without it, from what we saw.

Reply:
@sct It didn't seem to be the case when I tested, but let me do some more testing to verify.

Reply:
@sct I verified that it does respect the region without `with_release_type` being set.  `with_release_type` is just an additional filter so that returned results must have at least one release of the specified types in the region.  Without `with_release_type`, results are just required to have a release of any type for the region.

Setting this value to `3|2` just made the region setting appear to do more, since it limited the results to items with a theatrical release in the specified region.  I don't think this is desirable, since it filters out items that are exclusive to streaming services, for example.